portal-frontend/src/app/features/edit-submission/edit-submission.component.ts

```diff
@@ -247,9 +249,19 @@ export class EditSubmissionComponent implements OnInit, OnDestroy, AfterViewInit
   }
 
   async onSubmit() {
-    if (this.applicationSubmission) {
-      await this.applicationSubmissionService.submitToAlcs(this.applicationSubmission.uuid);
-      await this.router.navigateByUrl(`/application/${this.applicationSubmission?.fileNumber}`);
+    const submission = this.applicationSubmission;
+    if (submission) {
+      await this.applicationSubmissionService.submitToAlcs(submission.uuid);
+
+      const primaryContact = submission.owners.find((owner) => owner.uuid === submission?.primaryContactOwnerUuid);
+      if (primaryContact && primaryContact.type.code === APPLICATION_OWNER.GOVERNMENT) {
+        const review = await this.applicationReviewService.startReview(submission.fileNumber);
+        if (review) {
+          await this.router.navigateByUrl(`application/${submission?.fileNumber}/review`);
+        }
+      } else {
+        await this.router.navigateByUrl(`/application/${submission?.fileNumber}`);
+      }
```

Comment:
slash and no slash in routes, is it intentional?

Reply:
Added a slash here, but it seems like we are 70/30 with 30% not having a slash.